data-science-onramp/data-cleaning/clean.sh

```diff
@@ -0,0 +1,12 @@
+# Submit a PySpark job via the Cloud Dataproc Jobs API
+# Requires having PROJECT_ID, CLUSTER_NAME and BUCKET_NAME set as 
+# environment variables
+
+export CLUSTER_NAME=data-cleaning
+export PROJECT_ID=data-science-onramp
+export BUCKET_NAME=citibikevd
```

Comment:
Actually for cleaning I think we should have the user actually type the gcloud command themselves (or use the Google Cloud Console). For the setup we make a bash script only because we wanted to do things behind the scene to prepare the data for the tutorial.

Reply:
^Agreed! We do not need this script at all